packages/beacon-node/src/chain/lightClient/index.ts

```diff
@@ -494,21 +508,32 @@ export class LightClientServer {
         finalizedHeader &&
         (!this.finalized ||
           finalizedHeader.slot > this.finalized.finalizedHeader.slot ||
-          sumBits(syncAggregate.syncCommitteeBits) > sumBits(this.finalized.syncAggregate.syncCommitteeBits))
+          syncAggregateParticipation > sumBits(this.finalized.syncAggregate.syncCommitteeBits))
       ) {
         this.finalized = {
           attestedHeader: attestedData.attestedHeader,
           finalizedHeader,
           syncAggregate,
           finalityBranch: attestedData.finalityBranch,
+          signatureSlot,
         };
-        this.emitter.emit(ChainEvent.lightclientFinalizedUpdate, this.finalized);
         this.metrics?.lightclientServer.onSyncAggregate.inc({event: "update_latest_finalized_update"});
       }
     }
 
+    // Emit update
+    // messages SHOULD be broadcast after one-third of slot has transpired
+    // https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/light-client/p2p-interface.md#sync-committee
+    const minPubTime = computeTimeAtSlot(this.config, signatureSlot, genesisTime) + this.config.SECONDS_PER_SLOT / 3;
+    const waitTime = minPubTime - Date.now() / 1000;
+    await sleep(waitTime);
```

Comment:
This sleep shouldn't happen here, should happen where you had it before. We don't want the pause in the beacon API. We only need it for p2p.